From f6613905110827f71a2f55f1b1d0f1f71560023d Mon Sep 17 00:00:00 2001
From: Jin Qian <jinqian@google.com>
Date: Mon, 11 May 2015 17:57:52 -0700
Subject: [PATCH] MA-7240-3 KernelUidCpuTimeReader exception in mm6.0 Fix the
 uid permission error.

proc: uid_cputime: fix show_uid_stat permission

Change-Id: Ice9084e39da599261df0be6dc305b817b50cfbbf
Signed-off-by: Jin Qian <jinqian@google.com>
---
 drivers/misc/uid_cputime.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/uid_cputime.c b/drivers/misc/uid_cputime.c
index acd7046..c3f5bda 100644
--- a/drivers/misc/uid_cputime.c
+++ b/drivers/misc/uid_cputime.c
@@ -226,7 +226,7 @@ static int __init proc_uid_cputime_init(void)
 	proc_create_data("remove_uid_range", S_IWUGO, parent, &uid_remove_fops,
 					NULL);
 
-	proc_create_data("show_uid_stat", S_IWUGO, parent, &uid_stat_fops,
+	proc_create_data("show_uid_stat", S_IRUGO, parent, &uid_stat_fops,
 					NULL);
 
 	profile_event_register(PROFILE_TASK_EXIT, &process_notifier_block);
-- 
1.8.0

